index.js

```diff
@@ -2,5 +2,9 @@
 'use strict';
 
 module.exports = {
-  name: 'ember-cli-textarea-autosize'
+  name: 'ember-cli-textarea-autosize',
+  included: function (app) {
+    this._super.included(app);
+    app.import(app.bowerDirectory + '/autosize/dist/autosize.js');
```

Comment:
This directive in the addon itself tells the app that this addon is being used in to include the `autosize.js` library in the `vendor.js` file that broccoli is cooking.